### DISCUSSION TITLE
How do I see all the projects using the API

### DISCUSSION THREAD

**Discussion by dualarrow** [exceptionless/Exceptionless#1093] [Q&A] [answered]
I have a user with 3 organisation with a total of 20 projects. When I call the API with

/api/v2/projects

with a user scoped token, I only get 10 listed. I also noticed when I go to the web UI in the Settings/Projects, I see 10 on the 1st page and 10 on the second page. I'm guessing the 10 is either coincidental or theres a paging option I can use in the REST API.

Should I see more than the 10, or if there is a paging option, how do I define the next page in the /api/v2/projects GET ?

**Comment by niemyjski** [comment, marked as answer]
Hello,

You'd use the limit query string parameter as described in in our [api documentation](https://api.exceptionless.io/docs/index.html):
